src/translations/en.json

```diff
@@ -77,6 +79,20 @@
       }
     },
     "moves":{
+      "disabledMove" : {
+        "topTwo": "You must play one of the top two cards from the deck",
+        "notTurn": "It's not your turn",
+        "frozenCard": "This card is frozen"
+      },
+      "points": {
+        "displayName": "Points",
+        "description": "Gain 1 point | Gain {count} points"
```

Comment:
TIL. That's a pretty slick move and conveniently cleaner than the variable we had before applying localization as well. Nice!